sssd_test_framework/utils/authentication.py

```diff
@@ -422,6 +422,7 @@ def passkey_with_output(
             set timeout {DEFAULT_AUTHENTICATION_TIMEOUT}
             set prompt "\n.*\[#\$>\] $"
             set command "{command}"
+            set pin "{pin}"
```

Comment:
If pin is None the value would be `"None"`,

Reply:
Here, I consider the value as a `set pin ""`

Reply:
Yes, I know you consider it. But reality IS different. Try:

```python
val = None
print(f'set pin "{val}"')
```

Reply:
updated, Thanks!